change/@ni-nimble-components-1d7d08c5-fc96-4796-bce3-650510487895.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
@atmgrifter00 @jattasNI I can't remember, do we treat breaking changes on page objects as breaking api semver releases?

Reply:
Yeah, we should treat it as part of our public API. So mark this `major` and add a note to the `comment` saying the old and new name of the method.

If we want to fudge it we could claim that the whole grouping feature is still "internal" / "incubating" since it's only been around for a couple weeks and no one is using it yet. But that'd be kind of a made up policy and there's no harm in major bumps so I say we do a major bump.